config.md

```diff
@@ -41,17 +41,24 @@ Each container has exactly one *root filesystem*, specified in the *root* object
 
 You MAY add array of mount points inside container as `mounts`.
 The runtime MUST mount entries in the listed order.
-The parameters are similar to the ones in [the Linux mount system call](http://man7.org/linux/man-pages/man2/mount.2.html).
+The parameters are similar to the ones in [the Linux mount system call][mount.2].
 
 * **`destination`** (string, required) Destination of mount point: path inside container.
   For the Windows operating system, one mount destination MUST NOT be nested within another mount (e.g., c:\\foo and c:\\foo\\bar).
-* **`type`** (string, required) The filesystem type of the filesystem to be mounted.
-  Linux: *filesystemtype* argument supported by the kernel are listed in */proc/filesystems* (e.g., "minix", "ext2", "ext3", "jfs", "xfs", "reiserfs", "msdos", "proc", "nfs", "iso9660").
+* **`type`** (string, optional) The type of filesystem to mount.
+  If `type` is unset, the runtime MAY ask the kernel to guess the desired type.
+  Depending on the mount `options`, the `type` field MAY be ignored.
+  For example, `type` is ignored when `options` contains `bind` or `rbind`; see the `MS_BIND` description in [mount(2)][mount.2].
```

Comment:
On Wed, Aug 31, 2016 at 07:37:32AM -0700, Aleksa Sarai wrote:
> > +  For example, `type` is ignored when `options` contains `bind` or `rbind`; see the `MS_BIND` description in [mount(2)][mount.2].
>
> Shouldn't this be `recursive` and `bind`? Or am I misunderstanding
> what the purpose of the change is?

It shouldn't be mentioning ‘rbind’, so I've removed that with 1ef5938
→ 05b3277.  I'm sure ‘type’ is ignored for MS_BIND.  I'm less sure
it's always ignored for MS_REC, so I don't mention ‘recursive’ here in
the 05b3277 version of this line.  I can't think of a case where you'd
specify MS_REC *and* a filesystem type, but we only need a single
example of ignoring ‘type’ here and don't need to be exhaustive.